Makefile

```diff
@@ -198,6 +199,10 @@ cmd/grafana-agent-crow/grafana-agent-crow: cmd/grafana-agent-crow/main.go
 	$(ALL_CGO_BUILD_FLAGS) ; $(seego) build $(CGO_FLAGS) -o $@ ./$(@D)
 	$(NETGO_CHECK)
 
+cmd/agent-smoke/agent-smoke: seego cmd/agent-smoke/main.go
```

Comment:
In general, I also think it was a mistake for me to put crow in `cmd/`, and I think both crow and this should probably be in tools instead. These are things we wrote for development, and don't want to put them in the folder that signals "please use me"